Assets/Resources/Post-Processing Profile/CC.asset

```diff
@@ -77,7 +77,7 @@ MonoBehaviour:
       useCameraFov: 0
       kernelSize: 1
   motionBlur:
-    m_Enabled: 1
```

Comment:
revert this @DashaShev 
You can disable this option in your Camera but don't apply it on prefab because on all cameras this option changes